test/e2e/persistentips_test.go

```diff
@@ -311,3 +341,67 @@ func removeFinalizersPatch() ([]byte, error) {
 	}
 	return json.Marshal(patch)
 }
+
+func secondaryNetworkVMIStatusIPs(vmi *kubevirtv1.VirtualMachineInstance) ([]string, error) {
+	return testenv.GetIPsFromVMIStatus(vmi, secondaryLogicalNetworkInterfaceName), nil
+}
+
+func defaultNetworkStatusAnnotationIPs(vmi *kubevirtv1.VirtualMachineInstance) ([]string, error) {
+	defNetworkStatus, err := testenv.DefaultNetworkStatus(vmi)
+	if err != nil {
+		return nil, err
+	}
+
+	return defNetworkStatus.IPs, nil
+}
+
+func vmiWithMultus(namespace string) *kubevirtv1.VirtualMachineInstance {
+	interfaceName := secondaryLogicalNetworkInterfaceName
+	return testenv.NewVirtualMachineInstance(
+		namespace,
+		testenv.WithMemory("128Mi"),
+		testenv.WithInterface(kubevirtv1.Interface{
+			Name: interfaceName,
+			InterfaceBindingMethod: kubevirtv1.InterfaceBindingMethod{
+				Bridge: &kubevirtv1.InterfaceBridge{},
+			},
+		}),
+		testenv.WithNetwork(kubevirtv1.Network{
+
+			Name: interfaceName,
+			NetworkSource: kubevirtv1.NetworkSource{
+				Multus: &kubevirtv1.MultusNetwork{
+					NetworkName: nadName,
+				},
+			},
+		}),
+	)
+}
+
+func vmiWithPasst(namespace string) *kubevirtv1.VirtualMachineInstance {
+	const (
+		interfaceName        = "passtnet"
+		cloudInitNetworkData = `
+version: 2
+ethernets:
+  eth0:
+    dhcp4: true`
```

Comment:
btw this is not required as far as i remember for our machines right ?
going to try and drop it during the effort to convert to managedTap